docs/formats/zson.md

```diff
@@ -52,11 +52,11 @@ For union values, multiple decorators might be
 required to distinguish the union-member type from the possible set of
 union types when there is ambiguity, as in
 ```
-123 (float64) (int64,float64)
+123.0 (float32) ((int64,float32,float64))
```

Comment:
ZSON doesn't print the 0 here:
```suggestion
123. (float32) ((int64,float32,float64))
```

Reply:
@mattnibs: Indeed, the current ZSON implementation in the Zed tooling does not print the trailing `0`. At times I've heard @mccanne make the distinction between what the spec says and what our implementation does and imply they don't always have to match 100%. For instance, there's an existing example already in the text just a little further down that shows it with the `123.0` so I assumed that was intentional/harmless and left it intact, hence made the example here match that one.  If @mccanne or @nwt have an opinion perhaps they can chime in. If not, I'm happy to change both to just `123.`. so they both match the current implementation.

Reply:
I checked in with @nwt to see if he had an opinion. He would have been fine leaving the trailing zeros but is also fine dropping them. Therefore I've dropped in both examples so they both now say `123.`.